CMakeLists.txt

```diff
@@ -102,3 +108,4 @@ ADD_SUBDIRECTORY(src)
 ADD_SUBDIRECTORY(tests)
 
 PKG_CONFIG_APPEND_LIBS(${PROJECT_NAME})
+PKG_CONFIG_APPEND_BOOST_LIBS(${BOOST_COMPONENTS_LIB})
```

Comment:
The generated .pc will change depending on `BUILD_TESTING`, including `unit_test_framework` (I guess as a required thing) or not.

In this case, I don't think it really matter, but in the general case, it would be nice to have a workflow where test dependencies doesn't propagate to dependent projects.

Reply:
BOOST_COMPONENTS_LIB never contains *unit_test_framework* so it should never appear in the pc file.

Reply:
I totally misread that, you're right,  thanks !